Helper/Entity/ProductHelper.php

```diff
@@ -301,8 +301,10 @@ public function setSettings($indexName, $indexNameTmp, $storeId, $saveToTmpIndic
         $indexSettings = $transport->getData();
 
         $this->algoliaHelper->setSettings($indexName, $indexSettings, false, true);
+        $this->logger->log('Settings: ' . json_encode($indexSettings));
```

Comment:
How about adding the index name here ?

Reply:
The index name is already logged here: https://github.com/algolia/algoliasearch-magento-2/pull/675/files#diff-55d7adf73efa24c6e154bba2a58b6d58R201